velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -36,53 +34,179 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
     parquet::registerParquetReaderFactory();
   }
 
+  void assertSelect(
+      std::vector<std::string> outputColumnNames,
+      const std::string& sql) {
+    std::vector<TypePtr> types = getTypes(outputColumnNames);
+
+    auto plan =
+        PlanBuilder()
+            .tableScan(ROW(std::move(outputColumnNames), std::move(types)))
+            .planNode();
+
+    assertQuery(plan, splits_, sql);
+  }
+
+  void assertSelectWithFilter(
+      std::vector<std::string>&& outputColumnNames,
+      common::test::SubfieldFilters filters,
+      const std::string& sql) {
+    std::vector<TypePtr> types = getTypes(outputColumnNames);
+    auto rowType = ROW(std::move(outputColumnNames), std::move(types));
+
+    auto plan = PlanBuilder()
+                    .tableScan(
+                        rowType,
+                        makeTableHandle(std::move(filters)),
+                        allRegularColumns(rowType))
+                    .planNode();
+
+    assertQuery(plan, splits_, sql);
+  }
+
+  void assertSelectWithAgg(
+      std::vector<std::string>&& outputColumnNames,
+      const std::vector<std::string>& aggregates,
+      const std::vector<ChannelIndex>& groupingKeys,
+      const std::string& sql) {
+    std::vector<TypePtr> types = getTypes(outputColumnNames);
```

Comment:
getTypes return is always used to make a RowType. Consider folding that logic in, e.g. 

```
auto rowType = toRowType(outputColumnNames);
...
.tableScan(rowType)
```